samples/GraphQL.DataLoader.Example/GraphQl/DealerShipQuery.cs

```diff
@@ -0,0 +1,17 @@
+﻿using GraphQL;
+using GraphQL.Types;
+
+namespace DataLoaderGql.GraphQl;
+
+public class DealerShipQuery : ObjectGraphType
+{
+    public DealerShipQuery()
+    {
+        Field<SalesmanGraphType>("salespeople").Argument<string>("name").AllowAnonymous().Resolve(ctx =>
```

Comment:
each chained method should be on new line.  `AllowAnonymous` is part of authentication and out of scope for this example.